practice-one/src/components/Gender/Gender.css

```diff
@@ -0,0 +1,17 @@
+.gender__wrapper {
+  display: inline-block;
```

Comment:
please make sure <p> tag is not block scope

Reply:
I made it base on bootstrap badges
![image](https://github.com/datdragon777/React-Training/assets/74806826/9853f545-8630-47fb-9369-c364dcc3813c)